api/http/handler/users/user_inspect.go

```diff
@@ -15,6 +15,7 @@ import (
 // @id UserInspect
 // @summary Inspect a user
 // @description Retrieve details about a user.
+// @description User passwords are filtered out, and should never be accessible.
```

Comment:
We are returning  portainer.User from this line below;
`// @success 200 {object} portainer.User {Success}`
And that includes the password field.  What you need is to add a swagignore option.  See below.

Reply:
So this is why i removed the `example:""` - I'm expecting that it'll thus elide it from the general purpose example

whereas the update user code has its own input struct that contains an example password value.

Reply:
yup, I agree, and the `swaggerignore` works, and doesn't elide it from the other (create, update password) use, because those duplicate the struct with different values.
I'm going to leave the English in, its worth re-iterating the intent of the magic.